### DISCUSSION TITLE
Support usage of markdown links instead of [wikilinks]

### DISCUSSION THREAD

**Discussion by dvcrn** [notaapp/talk#157] [Feature Requests] [answered]
There are situations when it makes sense to use markdown links instead of wikilinks, like when the folder is published to github or other markdown-based systems. Right now Nota can do autocomplete for wikilinks when starting with`[[` but it doesn't have autocomplete for normal markdown links

It would be nice if nota could generate a normal markdown link, so `[abc](./foo.md)` instead of the wikilink when using the `[[` autocomplete

(This is a feature stolen from Obsidian but it's a very useful setting to have)

**Comment by astoilkov** [comment]
I agree. I've implemented it and the 0.35 release will include this feature.

Thanks!

**Reply by dvcrn** [reply]
@astoilkov any idea when we'll get 0.35? Would love to use github links over wikilinks

**Comment by astoilkov** [comment]
Tomorrow or the day after that, we will start testing a release candidate. If everything works, you can expect it next week, if not, the week after.

**Reply by dvcrn** [reply]
Hey @astoilkov! I saw 0.35 is out now. Was just curious if this feature made it in

**Reply by astoilkov** [reply]
Yes, it did. You can go to Preferences → Workspace → Use wiki links and uncheck it. The setting is per workspace so, for example, you can use the Markdown link syntax in your blog and use wiki links in your notes workspace.

**Comment by astoilkov** [comment, marked as answer]
This feature is now available in 0.35.

It can be enabled by unchecking Preferences → Workspace → Use wiki links. The setting is per workspace so, for example, you can use the Markdown link syntax in your blog and use wiki links in your notes workspace.

**Comment by davistech** [comment]
How do I get 0.35 (I have 0.34)?

**Reply by astoilkov** [reply]
The update is being gradually made available to users. We will make it available to everyone on Monday.
